profiles/seccomp/seccomp_default.go

```diff
@@ -254,6 +254,7 @@ func DefaultProfile() *types.Seccomp {
 				"renameat",
 				"renameat2",
 				"restart_syscall",
+				"rseq",
```

Comment:
Can you update the JSON file accordingly as well? You can do so using; https://github.com/moby/moby/blob/0d496e3d71ed02c05e47104781aa66c034af0264/profiles/seccomp/seccomp.go#L16

Or edit manually

Reply:
Oh, and can you keep this list sorted? `rseq` should probably be below `rmdir`